examples/gamma_estimate.csv

```diff
@@ -1,2 +1,2 @@
 estimate,std_err
-3.329153904463066,2.0256286444979983
+1.8450286322049927,0.3898786195682134
```

Comment:
Pretty meaningful change here. Is this just luck of the (random) draw?

Reply:
OK! It's anyway just a hypothetical example. Just want to have some confidence we know how we're generating it.